manifests/init.pp

```diff
@@ -837,6 +838,19 @@
   validate_re($sshd_pam_access, $valid_pam_access_values,
     "pam::sshd_pam_access is <${sshd_pam_access}> and must be either 'required', 'requisite', 'sufficient', 'optional' or 'absent'.")
 
+  case type($limits_fragments_hiera_merge) {
```

Comment:
The type() function is deprecated as `type` is now a reserved word. Check out the is_* functions.

https://github.com/puppetlabs/puppetlabs-stdlib/tree/4.2.0#is_string